### ISSUE TITLE
Alt text of the media on the self hosted sites is not visible in Media Browser

### ISSUE THREAD

**Issue by khaykov** [wordpress-mobile/WordPress-iOS#10216]
Corresponding android [issue](https://github.com/wordpress-mobile/WordPress-Android/issues/8393).

Steps to reproduce:

1. On self-hosted website navigate to media browser.
2. Open media settings of an image.
3. Add Alt text.

[![Image from Gyazo](https://i.gyazo.com/a736bc733a28691c13c05d29830bfc42.png)](https://gyazo.com/a736bc733a28691c13c05d29830bfc42)

 4. Open the media details of same image in the App.
 5. Notice that Alt text is missing.

[![Image from Gyazo](https://i.gyazo.com/6eebbb04b5e6d28724fdf0e10cc9f0b0.png)](https://gyazo.com/6eebbb04b5e6d28724fdf0e10cc9f0b0)

Screenshots are from Android.

**Comment by joashrajin** [comment]
I can confirm that this issue is still occurring in JP 22.4 & WPMobile 22.4. 

WP-Admin Media Library:
<img width="626" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/16253818/525666b4-e99e-4bed-a3d3-8322aa889ee9">

JP Mobile App:
<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/16253818/ea205fc0-93ce-47d9-8058-462f20f3e080" width="320px">

**Comment by tiagomar** [comment]
Related Android issue: https://github.com/wordpress-mobile/WordPress-Android/issues/18514

**Comment by kean** [comment]
Looks like this was fixed https://core.trac.wordpress.org/ticket/58582, is it should be ready for development.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
